assets/settings/default.json

```
@@ -303,6 +303,11 @@
     // Whether to show the Selections menu in the editor toolbar
     "selections_menu": true
   },
+  // Titlebar related settings
+  "title_bar": {
+    // Whether to show the branch icon beside branch switcher in the title bar.
+    "show_branch_icon": true
```

```suggestion
    "show_branch_icon": false
```

Based on the previous review, we'd prefer to keep this disabled by default.

---

Done !